.github/workflows/validate.yml

```
@@ -465,24 +465,21 @@ jobs:
     needs: [validate, validate-old-ghcs, build-alpine, dogfooding]
 
     steps:
-      - uses: liudonghua123/delete-release-action@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          release_name: 'cabal-head'
-
       - uses: actions/download-artifact@v4
         with:
           pattern: cabal-*
           path: binaries
           merge-multiple: true
 
-      - name: Create GitHub prerelease
-        uses: softprops/action-gh-release@v2
+      - name: (Re)Create GitHub prerelease
+        uses: andelf/nightly-release@main
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           tag_name: cabal-head
+          name: 'Cabal Nightly Release $$'
```

```suggestion
          name: cabal-head
```

We never called it nightly, so I'd prefer we stick to the existing terminology. And it doesn't happen every night.

---

Sure i'll update it